platform/darwin/src/NSComparisonPredicate+MGLAdditions.mm

```diff
@@ -318,14 +318,32 @@ - (id)mgl_jsonExpressionObject {
         case NSNotEqualToPredicateOperatorType:
             op = @"!=";
             break;
+        case NSBetweenPredicateOperatorType: {
+            op = @"all";
+            NSArray *arguments = self.rightExpression.constantValue;
```

Comment:
This variable name is misleading. The right expression contains the minimum and maximum bounds, so `bounds` or `limits` would be a better name.